pkgs/development/libraries/libdeltachat/default.nix

```diff
@@ -16,33 +16,25 @@
 
 stdenv.mkDerivation rec {
   pname = "libdeltachat";
-  version = "1.60.0";
+  version = "1.63.0";
 
   src = fetchFromGitHub {
     owner = "deltachat";
     repo = "deltachat-core-rust";
     rev = version;
-    sha256 = "1agm5xyaib4ynmw4mhgmkhh4lnxs91wv0q9i1zfihv2vkckfm2s2";
+    sha256 = "1511jh38h7nmn4dpyi1vfxvhybcacc6gavwfifxvb5npirphziga";
   };
 
   patches = [
     # https://github.com/deltachat/deltachat-core-rust/pull/2589
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
done